docs/source/Getting-Started/Tutorials/Build-Enclave-App-Hello-World.rst

```diff
@@ -0,0 +1,257 @@
+Tutorial 2: Hello, World!
+======================================
+
+This tutorial explains how to build and run a simple "hello world" enclave.
+An enclave consists of an eapp and a runtime, but also needs the host that initializes and launches
+the enclave.
+Thus, each enclave source tree contains at least the host and eapp.
+
+Before jumping into the tutorial, please complete :doc:`Quick Start
+<../Running-Keystone-with-QEMU>`.
+
+
+Prerequisite
+------------------------------
+
+`musl-libc <https://www.musl-libc.org/>`_ is a lightweight standard library for replacing ``glibc``.
```

Comment:
Eyrie doesn't support printf, it can (optionally)
   support some I/O operations. OK to clarify that this lets things
   like printf print _unsafely_
   - Do we want to encourage using musl-libc's musl-gcc or the rv8
     toolchain? (which supports C++ and is 'nicer' sort of?)
   - A more general question is how we want to handle using libc's in
     general... For the tutorial we may want to stick with glibc (and
     have a separate doc discussing musl)

Reply:
does glibc work??

Reply:
if it works, we can talk about glibc here, and then add musl-libc later.

Reply:
glibc works (as much as musl does at least)